src/k8s_client/k8s_client.go

```diff
@@ -141,3 +146,39 @@ func (c *k8sClient) GetConfigMap(namespace string, name string) (*v1.ConfigMap,
 	}
 	return cm, nil
 }
+
+func (c *k8sClient) GetPods(namespace string, labelMatch map[string]string) ([]v1.Pod, error) {
+	listOptions := metav1.ListOptions{}
+	if labelMatch != nil {
+		labelSelector := metav1.LabelSelector{MatchLabels: labelMatch}
+		listOptions.LabelSelector = labels.Set(labelSelector.MatchLabels).String()
+	}
+
+	pod, err := c.client.CoreV1().Pods(namespace).List(context.TODO(), listOptions)
+	if err != nil {
+		return nil, err
+	}
+
+	return pod.Items, nil
+}
+
+func (c *k8sClient) GetPodLogs(namespace string, podName string, sinceSeconds int64) (string, error) {
```

Comment:
I think that this function should be renamed get MCs log, so that later we will not use it for other logs. MCS logs are small, so there is not problem to copy them to buffer. but other component logs can be huge

Reply:
but it can bring every log. i want it generic. If we will have something huge, will need to check how to use it another way, but it will be good for most of the pods